src/category_theory/functor_category.lean

```diff
@@ -65,6 +65,14 @@ infix ` ◫ `:80 := hcomp
 @[simp] lemma hcomp_app {H I : D ⥤ E} (α : F ⟶ G) (β : H ⟶ I) (X : C) :
   (α ◫ β).app X = (β.app (F.obj X)) ≫ (I.map (α.app X)) := rfl
 
+@[simp] lemma hcomp_id_app {H : D ⥤ E} (α : F ⟶ G) (X : C) : (α ◫ 𝟙 H).app X = H.map (α.app X) :=
+begin
+  simp only [id_app, hcomp_app],
+  apply id_comp,
```

Comment:
Does it work to put `id_comp` in the `simp` arguments here?

Reply:
This doesn't work for some reason.

Reply:
Frustrating, fair enough!

Reply:
```
by { dsimp, simp, } -- See note [dsimp, simp].
```
suffices.

Reply:
Done!